.travis.yml

```diff
@@ -64,12 +73,15 @@ before_script:
 
 script:
     - mkdir -p $build_path/girder_testing_build
-    - cd $build_path/girder_testing_build
-    - cmake -DPYTHON_COVERAGE:BOOL=${PY_COVG} -DPYTHON_VERSION:STRING=${TRAVIS_PYTHON_VERSION} -DCOVERAGE_MINIMUM_PASS=30 $girder_path
+    - cmake -DPYTHON_COVERAGE:BOOL=${PY_COVG} -DPYTHON_VERSION:STRING=${TRAVIS_PYTHON_VERSION} -DCOVERAGE_MINIMUM_PASS=40 -DJS_COVERAGE_MINIMUM_PASS=9 $girder_path
     - make
-    - gunzip -c data/plugins/girder_db_items/testdb.sql.gz | psql -U postgres sampledb
-    # Show that we loaded the database properly
+    # Load postgres and show that we loaded the database properly
+    - gunzip -c data/plugins/database_assetstore/testdb.sql.gz | psql -U postgres sampledb
     - psql -U postgres -h 127.0.0.1 sampledb -c "SELECT table_type,table_name FROM information_schema.tables WHERE table_schema='public' order by table_name;" | cat
+    # Load mongo and show that we loaded the database properly
+    - bunzip2 -c data/plugins/database_assetstore/mongodb.permits.json.bz2 | mongoimport --db boston --collection permits
+    - mongo boston --eval 'db.permits.count()'
+
     # Only some functions are available in the version that Travis has.
     # - psql -U postgres -h 127.0.0.1 sampledb -c "SELECT lower(proname), provolatile FROM pg_proc order by lower(proname);" | cat
-    - JASMINE_TIMEOUT=15000 ctest -VV -R '(coverage_reset|py_coverage_combine|girder_db_items|py_coverage$|py_coverage_xml)'
+    - JASMINE_TIMEOUT=15000 ctest -VV -R '(coverage_reset|coverage_combine|database_assetstore|coverage$|coverage_xml)'
```

Comment:
We recently added some handy config vars to girder to avoid these sorts of regexes. You can add the following config-time definitions:

    -DRUN_CORE_TESTS:BOOL=OFF -DBUILD_JAVASCRIPT_TESTS:BOOL=OFF -DTEST_PLUGINS:STRING=database_assetstore